types/stylelint/index.d.ts

```diff
@@ -915,10 +914,6 @@ declare namespace stylelint {
 		 * The column of the exclusive end position of the warning.
 		 */
 		endColumn?: number;
-		/**
-		 * The `EditInfo` object of autofix. This property is undefined if this message is not fixable.
-		 * @experimental
-		 */
 		fix?: EditInfo;
```

Comment:
[suggestion] The comment itself still seems helpful:

```suggestion
		/**
		 * The `EditInfo` object of autofix. This property is undefined if this message is not fixable.
		 */
		fix?: EditInfo;
```